### PR TITLE
docs(ci): update release steps with homebrew bump

### PR DESCRIPTION
**Pull request by jsjoeio**
This PR updates the `ci` docs with steps for updating the homebrew package more easily using `brew bump-forumla-pr`.

TODOS
- [x] address feedback

### CODE REVIEW

File: docs/FAQ.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ Please file all questions and support requests at https://github.com/cdr/code-se
 
 Please see [./ipad.md](./ipad.md).
 
-## Community projects (awesome-code-server)
+## Community Projects (awesome-code-server)
```
Comment (jsjoeio) [inline docs/FAQ.md:46]: `yarn fmt` was being weird so I capitalized Projects

File: docs/install.md [hunk 2]
```diff
@@ -14,7 +14,7 @@
 - [Standalone Releases](#standalone-releases)
 - [Docker](#docker)
 - [helm](#helm)
-- [App Engines (Azure, Heroku)](#app-engines)
+- [App Engines (Azure, Heroku)](#app-engines-azure-heroku)
```
Comment (jsjoeio) [inline docs/install.md:17]: Not sure what happened with that title link, but this should fix. automatically added by `yarn fmt`

File: ci/README.md [hunk 3]
```diff
@@ -43,8 +43,8 @@ Make sure you have `$GITHUB_TOKEN` set and [hub](https://github.com/github/hub)
 9. Update the AUR package.
    - Instructions on updating the AUR package are at [cdr/code-server-aur](https://github.com/cdr/code-server-aur).
 10. Wait for the npm package to be published.
-11. Update the homebrew package.
-    - Send a pull request to [homebrew-core](https://github.com/Homebrew/homebrew-core) with the URL in the [formula](https://github.com/Homebrew/homebrew-core/blob/master/Formula/code-server.rb) updated.
+11. Update the [homebrew package](https://github.com/Homebrew/homebrew-core/blob/master/Formula/code-server.rb).
+    1. If you're on macOS and have homebrew locally, run `brew bump-formula-pr --version=3.8.1 code-server` and update the version accordingly. This will bump the version and open a PR. Note: this will only work once the version is published on npm.
```
Comment (code-asher) [inline ci/README.md:47]: Thoughts on doing something like:
```
  1. Install homebrew (link to install docs)
  2. Run `brew bump-formula` etc etc
```

My thought process is that that generalizes the process to anything that you can install homebrew on (like Linux or WSL).

Comment (jsjoeio) [reply ci/README.md:47]: Genius! I couldn't remember if homebrew was macOS only

**Review by code-asher — APPROVED** [review]

**Comment by jsjoeio** [comment]
Note: the ci/linux-arm64 failed because it ran out of space.

**Pull request merged**